src/dftbp/api/mm/capi.F90

```diff
@@ -250,6 +250,29 @@ subroutine c_DftbPlus_processInput(handler, inputHandler, atomListHandler)&
 
   end subroutine c_DftbPlus_processInput
 
+  !> Obtain electrostatic potential in specified points
+  subroutine c_DftbPlus_get_potential(handler, n_locations, pot, locations)&
```

Comment:
I'd suggest to consistently add the term `elstat` or `ElStat` as a prefix to the potential.  (e.g. `c_DftbPlus_get_elstat_potential`, `instance%getElStatPotential`, etc.) That would be more specific and aligned with input keyword.